app/Models/Skill.php

```diff
@@ -73,14 +62,41 @@ public function classifications() // phpcs:ignore
         return $this->belongsToMany(\App\Models\Classification::class)->withTimestamps();
     }
 
+    // Version 2 application models.
+
+    public function experiences_work() // phpcs:ignore
+    {
+        return $this->morphedByMany(\App\Models\ExperienceWork::class, 'experience', 'experience_skills');
+    }
+
+    public function experiences_personal() // phpcs:ignore
+    {
+        return $this->morphedByMany(\App\Models\ExperiencePersonal::class, 'experience', 'experience_skills');
+    }
+
+    public function experiences_education() // phpcs:ignore
+    {
+        return $this->morphedByMany(\App\Models\ExperienceEducation::class, 'experience', 'experience_skills');
+    }
+
+    public function experiences_award() // phpcs:ignore
+    {
+        return $this->morphedByMany(\App\Models\ExperienceAward::class, 'experience', 'experience_skills');
+    }
+
+    public function experiences_community() // phpcs:ignore
+    {
+        return $this->morphedByMany(\App\Models\ExperienceCommunity::class, 'experience', 'experience_skills');
+    }
+
```

Comment:
We may want to add a relationship to ExperienceSkill as well, for completeness. It would be a simple one-to-many relationship:
```
public function experience_skills()
    {
        return $this->hasMany(\App\Model\ExperienceSkill::class);
    }
```